styles/main.css

```diff
@@ -0,0 +1,184 @@
+@import url("https://fonts.googleapis.com/css?family=Open+Sans|Pacifico");
+@import url("https://fonts.googleapis.com/css?family=Open+Sans");
+
+/*MY COLOR PALLET
```

Comment:
I'm impressed by your coding style all throughout your 20-ish files. I like that you use comments to describe your rationale and help human readers follow along with your code. I'm imagining your color pallet looks beautiful with the lynn2.jpg image.